test/server/cards/01-Core/ChaosPortal.spec.js

```diff
@@ -0,0 +1,54 @@
+describe('Chaos Portal', function() {
+    integration(function() {
+        describe('Chaos Portal\'s Ability', function() {
+            beforeEach(function() {
+                this.setupTest({
+                    player1: {
+                        house: 'logos',
+                        inPlay: ['chaos-portal'],
+                        discard: ['duskwitch', 'bad-penny']
+                    }
+                });
+            });
+
+            it('should prompt to choose a house', function() {
+                this.player1.useAction(this.chaosPortal);
+                expect(this.player1).toHavePrompt('Chaos Portal');
+                expect(this.player1.currentButtons).toContain('brobnar');
```

Comment:
expect(this.player1).toHavePromptButton('brobnar') is a helper method that exists that would be appropriate here

Reply:
Sweet - I'll change that up 👍. I have to admit I pretty much copied this straight from `Arise!` and didn't really look at it 😅.

Reply:
The framework has evolved over time, so it's entirely possible that this is a new thing, so don't worry :)